azurecompute-arm/src/main/java/org/jclouds/azurecompute/arm/util/BlobHelper.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jclouds.azurecompute.arm.util;
+
+import org.jclouds.ContextBuilder;
+import org.jclouds.azure.storage.domain.BoundedSet;
+import org.jclouds.azureblob.AzureBlobClient;
+import org.jclouds.azureblob.domain.BlobProperties;
+import org.jclouds.azureblob.domain.ContainerProperties;
+import org.jclouds.azureblob.domain.ListBlobsResponse;
+import org.jclouds.azurecompute.arm.domain.VMImage;
+import org.jclouds.util.Closeables2;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class BlobHelper {
+
+   public static void deleteContainerIfExists(String storage, String key, String containerName) {
+      final AzureBlobClient azureBlob = ContextBuilder.newBuilder("azureblob")
+              .credentials(storage, key)
+              .buildApi(AzureBlobClient.class);
+
+      try {
+         azureBlob.deleteContainer(containerName);
+      }
+      catch (Exception e) {
+         e.printStackTrace();
+      }
```

Comment:
This will not fail if the container does not exist (see [here](https://github.com/jclouds/jclouds/blob/master/providers/azureblob/src/main/java/org/jclouds/azureblob/AzureBlobClient.java#L196])). Should we better not catch any exception, leave just the try/finally and let the method fail if the container can't be deleted?